{{cookiecutter.project_slug}}/.github/workflows/pypi.yml

```diff
@@ -28,26 +28,17 @@ jobs:
       run: |
         python -m pip install cibuildwheel==1.6.4
 
-    - name: Install Visual C++ for Python 2.7
-      if: runner.os == 'Windows'
-      run: |
-        choco install vcpython27 -f -y
-
     - name: Build wheels
       env:
         # These variables allow you to customize the process how the wheels
         # for the Python packages are built. For a list of options, see this
         # page: https://cibuildwheel.readthedocs.io/en/stable/options/
         CIBW_BEFORE_BUILD: python -m pip install cmake
-        CIBW_SKIP: "*-win32"
+        CIBW_SKIP: "*-win32 cp27-* cp35-*"
```

Comment:
is there a reason to exclude win32 builds? I ask because it's not uncommon to have 32-bit python installations on windows

Reply:
This failed on the Hamming project and I had 0 incentive to investigate. While doing the cookiecutter, I pasted my script from Hamming in the first place. Also, I did not know that 32-bit installations are a thing. So, I reenable it and we investigate if it fails?

Reply:
On windows 32-bit python is still pretty common (e.g. the first windows download link on python.org is the 32-bit version, and pypy is only available as a 32-bit version on windows). So yes, let's see if it works.